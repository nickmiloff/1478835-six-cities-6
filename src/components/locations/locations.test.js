import React from 'react';
import {render} from '@testing-library/react';

import Locations from './locations';

describe(`Locations component test`, () => {
  it(`'Locations component' should render correctly`, () => {
    const activeLocation = `Paris`;

    const {container} = render(<Locations activeLocation={activeLocation} changeLocation={jest.fn()} />);

    expect(container).toMatchSnapshot();
  });
});
