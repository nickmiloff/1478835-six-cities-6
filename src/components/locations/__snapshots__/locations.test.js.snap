// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Locations component test 'Locations component' should render correctly 1`] = `
<div>
  <section
    class="locations container"
  >
    <ul
      class="locations__list tabs__list"
    >
      <li
        class="locations__item"
      >
        <a
          class="locations__item-link tabs__item tabs__item--active"
          data-testid="tab-link"
        >
          <span>
            Paris
          </span>
        </a>
      </li>
      <li
        class="locations__item"
      >
        <a
          class="locations__item-link tabs__item"
          data-testid="tab-link"
        >
          <span>
            Cologne
          </span>
        </a>
      </li>
      <li
        class="locations__item"
      >
        <a
          class="locations__item-link tabs__item"
          data-testid="tab-link"
        >
          <span>
            Brussels
          </span>
        </a>
      </li>
      <li
        class="locations__item"
      >
        <a
          class="locations__item-link tabs__item"
          data-testid="tab-link"
        >
          <span>
            Amsterdam
          </span>
        </a>
      </li>
      <li
        class="locations__item"
      >
        <a
          class="locations__item-link tabs__item"
          data-testid="tab-link"
        >
          <span>
            Hamburg
          </span>
        </a>
      </li>
      <li
        class="locations__item"
      >
        <a
          class="locations__item-link tabs__item"
          data-testid="tab-link"
        >
          <span>
            Dusseldorf
          </span>
        </a>
      </li>
    </ul>
  </section>
</div>
`;
